codebase/superdarn/src.bin/tk/tool/map_addhmb.1.17/map_addhmb.c

```diff
@@ -36,9 +36,9 @@
 struct CnvMapData *map[3];
 struct GridData *grd[3];
 
-float bndarr[24][26];
-float lathmb[26];
-int latcnt[26];
+float bndarr[24][36];
```

Comment:
```suggestion
float bndarr[24][36]; /* the boundary array sizes correspond to the grid within which the HMB boundary is searched for when determining the HMB from the data: [24 MLT hours] and [36 Geomagnetic Latitudes, ranging from 40 to 76 degrees] 
```

Reply:
This is in response to @aburrell 's suggestion for a comment here.

Reply:
Good informative comment, I support committing this change.

Reply:
I'm a fairly lay-person on the HMB and I fairly well understand why the array dimensions are that size.  So I'd say it's good to add in.